lib/src/helpers/extensions.dart

```diff
@@ -66,8 +67,8 @@ extension CanvasRenderingContext2DGlue on CanvasRenderingContext2D {
 
 extension NodeGlue on Node {
   set text(String s) => textContent = s;
-  void append(Node other) => appendChild(other);
-  void clone(bool deep) => cloneNode(deep);
+  Node append(Node other) => appendChild(other);
+  Node clone(bool? deep) => deep == null ? cloneNode() : cloneNode(deep);
```

Comment:
I believe we could rewrite this as `cloneNode(deep ?? false)`. The default value is [`false`](https://dom.spec.whatwg.org/#ref-for-dom-node-clonenode%E2%91%A0).

Reply:
Updated below:

```suggestion
  Node clone(bool? deep) => cloneNode(deep ?? false);
```

We may also be able to do `... => cloneNode(deep);` ?

Reply:
I actually don't know what passing `null` does! I assume it's considered "falsy" but there's no documentation there.